src/entry/worker.ts

```diff
@@ -49,6 +49,10 @@ parentPort!.on('message', (message: StartupMessage) => {
     const readyMessage: ReadyMessage = { ready: true }
     parentPort!.postMessage(readyMessage)
 
+    // On Bun we need to start the port explicitly, does not impact the Nodejs
+    // https://github.com/oven-sh/bun/issues/19863
+    port.start()
```

Comment:
We can remove the comment.

```suggestion
    port.start()
```